exportCategories.php

```diff
@@ -69,7 +69,12 @@ protected function exportData(Mage_Catalog_Model_Category $category, $file, $dep
         echo '* ' . $category->getName() . sprintf(' (%s products)', $category->getProductCount()) . PHP_EOL;
         fputcsv($file, $data);
         if ($category->hasChildren()) {
-            $children = Mage::getModel('catalog/category')->getCategories($category->getId());
+            $children = Mage::getModel('catalog/category')
+                ->load($category->getId())
```

Comment:
Hi adriancr,

thanks for your contribution!
It might be better to drop this load(), since you just grab the collection afterwards.
Also, is there a reason, why you sort the collection by position or might that be dropped, too?